x/observer/simulation/operations.go

```diff
@@ -0,0 +1,101 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/zeta-chain/node/x/observer/keeper"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+// Simulation operation weights constants
+// Operation weights are used by the simulation program to simulate the weight of different operations.
+// This decides what percentage of a certain type of operation is part of a block.
+// Based on the weights assigned in the cosmos sdk modules , 100 seems to the max weight used , and therefore guarantees that at least one operation of that type is present in a block.
```

Comment:
Any link regarding this? I might think weights are per modules, and the number doesn't matter if you have a single operation

Reply:
I think this comment might help 
https://github.com/zeta-chain/node/pull/3095#discussion_r1878077246